sdk/storage/README.md

```diff
@@ -2,6 +2,8 @@
 
 Azure Storage is a Microsoft-managed service providing cloud storage that is highly available, secure, durable, scalable, and redundant.  Azure Storage includes Blobs (objects), Queues, and Files.
 
+- [Azure.Storage][meta] This is our storage meta-package that references the most common storage libraries, for the time being it only references Blobs.
```

Comment:
Can we move this last after the other Azure.Storage packages?  I don't want folks to start here.